transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/EncodeTransformationPipelineBuilder.scala

```diff
@@ -85,6 +88,22 @@ object EncodeTransformationPipelineBuilder extends PipelineBuilder[Args] {
     ()
   }
 
+  /** Message value used by ENCODE in place of null. */
+  val UnknownValue: Msg = Str("unknown")
+
+  /**
+    * Process an arbitrary ENCODE object to strip out all
+    * values representing null / absence of data.
+    */
+  def removeUnknowns(rawObject: Msg): Msg = {
```

Comment:
this looks good to me; do you think it'd be good practice to make this extensible to add other things to strip out in the future (or pre-process in other ways here), or would that be overkill for now? Since this looks like it is set up to have just the one `UnknownValue` as opposed to a set of possible strings or something.

Reply:
I think it'd be overkill for now, but if/when we hit the next use case we should parameterize this one instead of making another hard-coded variant.